contrib/pkg/awsprivatelink/awsprivatelink.go

```diff
@@ -17,6 +17,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// privateLinkHubAcctCredsName is the name of the AWS PrivateLink Hub account credentials Secret
+	// created by the "hiveutil awsprivatelink enable" command
+	privateLinkHubAcctCredsName = "awsprivatelink-hub-acct-creds"
+
+	// privateLinkHubAcctCredsLabel is added to the AWS PrivateLink Hub account credentials Secret
+	// created by the "hiveutil awsprivatelink enable" command and
+	// referenced by HiveConfig.spec.awsPrivateLink.credentialsSecretRef.
+	privateLinkHubAcctCredsLabel = "hive.openshift.io/awsprivatelink-hub-acct-credentials"
+)
```

Comment:
only used in this package, so moved here from github.com/openshift/hive/contrib/pkg/utils/aws